canal/cyclone.py

```diff
@@ -401,7 +401,10 @@ def __init__(self, x: int, y: int, num_track: int, width: int):
 
 class ImranSwitchBox(SwitchBox):
     def __init__(self, x: int, y: int, num_track: int, width: int):
-        internal_wires = SwitchBoxHelper.get_imran_sb_wires(num_track)
+        if mod(num_track, 2)==1: # odd number of tracks
```

Comment:
Use pystylecode to lint the diff.

Reply:
done in the latest commit 9b1c6b7, can do a squash commit later